docs/sdkx_ios/sdk-configuration.mdx

```diff
@@ -337,6 +337,50 @@ Helpshift.showConversation(with:self, config:config)
 
 In your registration process, ask your user for his/her age. If the user's age is 13 or younger, set enableFullPrivacy to YES. This way, you comply with COPPA for your children users, but collect valuable user and device data for your other users.
 
+## initiateChatOnLoad {#initiateChatOnLoad}
+
+Upon setting `initiateChatOnLoad` to `YES`, when the conversation loads if the previous issue is resolved, a new conversation will be started, without end user clicking on the `New Conversation` button or going through the post resolution flows like feedback bots of previous issue.
```

Comment:
@rehanHS Please ensure this line reads the same in both iOS and Android docs. I would suggest to copy over the android line here since that is better structured.

Reply:
Okay